src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -391,7 +391,7 @@
       (->
         (if (some #(= queried-state %) ["requested" "signed" "revoked"])
```

Comment:
Found a very interesting bug (maybe?) here.  Apparently `queried-state` is `nil` when I ran the command `puppetserver-ca list` so the function `filter-by-certificate-state` is not getting called.  I checkout a fresh 6.x branch without any of my changes and ran the same command again and `filter-by-certificate-state` is not getting called as well.  I tried to log the value of `queried-state` before the conditional and it is always `nil`, with or without the `--all` flag.  So at of right now, any of my changes are not effective because of this conditional always being false.